### ISSUE TITLE
Choose/upload not working?

### ISSUE THREAD

**Issue by SHesg** [BarryFogarty/Umbraco.SimpleFileUpload#1]
Hi

New to umbraco and found your post on their forum which led me here. Can't seem to get the code to work though.

Followed the steps per your listing, and in the backoffice, the File Upload section brings up the app.html - but the choose file button doesn't seem to activate.

To me it looks like it isn't binding to the controller or activating, but not sure why. Any thoughts?

Regards
Sean

**Comment by BarryFogarty** [comment]
Are there any console errors when you load the dashboard control?  TBH This was done for an early v7 Umbraco version, not sure if it is compatible with the latest.  If I was approaching this task again I would try to re-use or extend the built-in file upload component from the media section.

**Comment by SHesg** [comment]
Hi Barry

No - there aren't any other errors. Section loads and I can see the buttons, just nothing happens I'm afraid.

Any pointers/articles that you've come across to put me on the right course re extending the upload component?

Regards
Sean

**Comment by BarryFogarty** [comment]
Thanks for the report, I'll install it on the latest version of Umbraco and see if I can repro.

**Comment by SHesg** [comment]
Cheers - let me know how it goes
Sean

**Comment by BarryFogarty** [comment]
Sean - I installed this on the latest Umbraco 7.6.1 and it works fine on Chrome and IE11.  I did notice that there was a console error: `Unknown provider: UploadProvider <- Upload <- umbFileDropzoneDirective`.  It seems there is a conflict with the in-built angular file upload module in Umbraco, which results in  dragn'drop of media breaking.  However the core functionality still works.  I am going to see if I can update the plugin to use the built-in file upload mechanism soon.
